src/components/Create/components/pages/ReconfigurationRules/ReconfigurationRulesPage.tsx

```diff
@@ -76,6 +76,12 @@ export const ReconfigurationRulesPage = () => {
             >
               <JuiceSwitch label={t`Allow terminal configuration`} />
             </Form.Item>
+            <Form.Item
+              name="pauseTransfers"
+              extra={t`When enabled, transfers will be paused.`}
```

Comment:
... project token transfers will be paused. This does not apply to ERC-20 tokens if issued.